.github/actions_scripts/analyse_git_reference.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import sys
+import os
+
+def get_jamulus_version(repo_path_on_disk):
+    jamulus_version = ""
+    with open (repo_path_on_disk + '/Jamulus.pro','r') as f:
+        pro_content = f.read()
+    pro_content = pro_content.replace('\r','')
+    pro_lines = pro_content.split('\n')
+    for line in pro_lines:
+        line = line.strip()
+        VERSION_LINE_STARTSWITH = 'VERSION = '
+        if line.startswith(VERSION_LINE_STARTSWITH):
+            jamulus_version = line[len(VERSION_LINE_STARTSWITH):]
+    return jamulus_version
+
+
+if len(sys.argv) == 1:
+    repo_path_on_disk = os.environ['GITHUB_WORKSPACE'] 
+    release_version_name = get_jamulus_version(repo_path_on_disk)
+    
+    os.system('perl "{}"/.github/actions_scripts/getChangelog.pl "{}"/ChangeLog "{}" > "{}"/autoLatestChangelog.md'.format(
+        os.environ['GITHUB_WORKSPACE'],
+        os.environ['GITHUB_WORKSPACE'],
+        release_version_name,
+        os.environ['GITHUB_WORKSPACE']
+    ))
+
+elif len(sys.argv) == 4:
+    #fullref=sys.argv[1]
+    #pushed_name=sys.argv[2]
+    release_version_name = sys.argv[3]
+else:
+    print('Expecing 4 arguments, 1 script path and 3 parameters')
+    print('Number of arguments:', len(sys.argv), 'arguments.')
+    print('Argument List:', str(sys.argv))
+    raise Exception("wrong agruments")
+    
+fullref=os.environ['GITHUB_REF']
+reflist = fullref.split("/", 2)
+pushed_name = reflist[2]
+
+
+
+if fullref.startswith("refs/tags/"):
+    print('this reference is a Tag')
+    publish_to_release = True
+    release_tag = pushed_name # tag already exists
+    release_title="Release {}  ({})".format(release_version_name, pushed_name)
+    if pushed_name.startswith("r"):
+        if "beta" in pushed_name:
+            print('this reference is a Beta-Release-Tag')
+            is_prerelease = True
+        else:
+            print('this reference is a Release-Tag')
+            publish_to_release = True
+            is_prerelease = False
+    else:
+        print('this reference is a Non-Release-Tag')
+        publish_to_release = False
+        is_prerelease = True # just in case
+    
+        
+    if pushed_name == "latest":
+        print('this reference is a Latest-Tag')
```

Comment:
Still to be discussed: do we want to move the latest tag automatically?

Reply:
this is safe behaviour for now: the latest tag is not moved automatically, but the release "latest" is recreated anew when the tag "latest" is moved.
github also has a separate "latest release" feature

I see the behaviour mainly part of #854.
My goal for now is to bring some version of android build in the main repo without breaking things

Reply:
The current action DOES move the latest tag automatically.

Reply:
the "latest" tag is NOT automatically moved.
Only the "latest" release is moved along, when the "latest" tag ist moved manually

Reply:
Sorry that I haven't been clear enough: the current action on master would move the latest tag.